File: Matlab/P001.m
providedNum = 1000;
mtpList = [];
for i = 1:providedNum-1
  if mod(i,3) == 0
    mtpList = [mtpList i];
  end
  if mod(i,5)==0 && mod(i,3)~=0
    mtpList = [mtpList i];
  end
end
sum(mtpList)

File: C++/P001.cpp
#include <iostream>

int main(void){
  int providedNum = 1000;
  int mtpSum = 0;

  for (int i=0; i<providedNum; i++){
    if (i%3 == 0)
      mtpSum += i;
    if (i%5 == 0 && i%3 != 0)
      mtpSum += i;
  }

  std::cout << mtpSum << std::endl;
}